packages/react-native-codegen/scripts/oss/build.sh

```diff
@@ -32,12 +32,24 @@ if [[ ${FBSOURCE_ENV:-0} -eq 1 ]]; then
   "$YARN_BINARY" run build
 
   popd >/dev/null
+
 else
   # Run yarn install in a separate tmp dir to avoid conflict with the rest of the repo.
   # Note: OSS-only.
   TMP_DIR=$(mktemp -d)
 
-  cp -R "$CODEGEN_DIR/." "$TMP_DIR"
+  # On Windows this script gets run by a seprate Git Bash instance, which cannot perform the copy
+  # due to file locks created by the host process. Need to exclude .lock files while copying.
+  # Using in-memory tar operation because piping `find` and `grep` doesn't preserve folder structure
+  # during recursive copying, and `rsync` is not installed by default in Git Bash.
+  # As an added benefit, blob copy is faster.
+  if [ "$OSTYPE" = "msys" ]; then
+    my_dir=`pwd`;
+    tar cf - --exclude=*.lock $CODEGEN_DIR | (cd $TMP_DIR && tar xvf - );
+    cd $my_dir;
```

Comment:
Since the pipe to `cd ...` above is run in a subshell, there should be no need to change back to the current working directory (it will not have changed)

Reply:
This whole script is run in a subshell. I wanted to change back to the directory we were in before line 48 for the sake of the rest of this script, rather than the code running this file.

Reply:
Oh, but then right after `cd $my_dir` there's a `pushd "$TEMP_DIR"` and I'm already there, so no need for the cd - is that what you meant?